### DISCUSSION TITLE
Setup Within a URL Subdirectory

### DISCUSSION THREAD

**Discussion by AccutarReg** [lukevella/rallly#698] [Q&A] [answered]
Is it possible to setup rallly using a URL Subdirectory like: http://[domain]/rallly

I tried setting the NEXT_PUBLIC_BASE_URL=http://[domain]/rallly
I am also using nginx to reverse proxy and configured it to /rallly. When I attempt to open it in a browser, I get a blank page. Is this a limitation in rallly or am I missing something? 

I appreciate any help with this!

**Comment by lukevella** [comment, marked as answer]
I’m pretty sure this isn’t possible due to the way the internal app router works. Using a subdomain is the way to go.

**Reply by AccutarReg** [reply]
Ah I see, thanks for the quick response!

**Comment by paalbra** [comment]
Is this still not possible?

I have zero knowledge of Next.js, and have tried to fumble around with variables.

The `basePath` seems to be meant for this, or am I wrong? https://nextjs.org/docs/pages/api-reference/next-config-js/basePath

The note says this though:

> Good to know: This value must be set at build time and cannot be changed without re-building as the value is inlined in the client-side bundles.

I'm using the [docker image](https://hub.docker.com/r/lukevella/rallly). I guess `basePath` won't work in this image since the application is already built?

I haven't tried this yet since, like I said, I have zero knowledge of Next.js.

**Reply by paalbra** [reply]
I suppose I could build the container with this Dockerfile? https://github.com/lukevella/rallly/blob/main/apps/web/Dockerfile

I'm not able to build it as is though. It fails early here:
```
turbo prune --scope=@rallly/web --docker
 WARNING  No locally installed `turbo` found. Using version: 2.3.3.
turbo 2.3.3

  x could not resolve workspaces
  `-> missing packageManager field in package.json
```
